jetson/odrive_bridge/src/__main__.py

```diff
@@ -234,14 +225,16 @@ def connect(self):
         # odrive 2 --> back motors
 
         odrives = ["335D36623539", "335B36563539", "335536553539"]
+
         id = odrives[legal_controller]
 
         print(id)
+        # import pdb; pdb.set_trace()
         odrive = odv.find_any(serial_number=id)
 
         print("found odrive")
         modrive = Modrive(odrive)  # arguments = odr
-        modrive.set_current_lim(100)
+        modrive.set_current_lim(4)

```

Comment:
Is this related to the CURRENT_LIM constant in Modrive? Could you use constant for these values instead of just setting them?

Reply:
resolved